### PR TITLE
Add hungarian Lego Mindstorms books

### PR DESCRIPTION
**Pull request by vicziani**
Add two hungarian books about programming Lego Mindstorms.

### CODE REVIEW

File: free-programming-books-hu.md [hunk 1]
```diff
@@ -45,13 +46,16 @@
 * [CORBA-alapú elosztott alkalmazások](http://mek.oszk.hu/01400/01404/index.phtml) - Csizmazia Balázs (PDF)
 * [Fantasztikus programozás](http://mek.oszk.hu/00800/00889/index.phtml) - Bátfai Mária Erika, Bátfai Norbert (PDF)
 * [Hálózati alkalmazások Java nyelven](http://mek.oszk.hu/01300/01304/index.phtml) - Csizmazia Anikó, Csizmazia Balázs (PDF)
-* [Hálózati alkalmazások készítése: CORBA, Java, WWW](http://mek.oszk.hu/01700/01750/index.phtml) - Csizmazia Balázs (PS) 
+* [Hálózati alkalmazások készítése: CORBA, Java, WWW](http://mek.oszk.hu/01700/01750/index.phtml) - Csizmazia Balázs (PS)
 * [Java alapú webtechnológiák](http://www.ms.sapientia.ro/~manyi/index_java_techn.html) - Antal Margit (PDF)
```
Comment (vhf) [inline free-programming-books-hu.md]: I cannot access the linked files, they all point to `#`.

Comment (vicziani) [inline free-programming-books-hu.md]: The links of the unavailable formats are grey which point to `#`. The PS and Java formats are downloadable, the links work, for example `http://mek.oszk.hu/01700/01750/01750ps.zip`. It's not obvious, but this is a site from the past century. :)

Comment (vhf) [inline free-programming-books-hu.md]: Ok. If the site is from the past century, is the content still relevant today?

Comment (vicziani) [inline free-programming-books-hu.md]: Yes, if you develop legacy applications. For example the Java RMI or CORBA have not changed in years. The last version of CORBA: December 2002. The last version of RMI: June 2002.

**Pull request merged**